### PR TITLE
Cleanup test/e2e/common directory - part2

### PR DESCRIPTION
**Pull request by wojtek-t**
Ref https://github.com/kubernetes/kubernetes/issues/98326

```release-note
NONE
```

/kind cleanup
/priority important-soon

### CODE REVIEW

File: test/e2e/common/storage/downwardapi.go [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/uuid"
+	"k8s.io/kubernetes/test/e2e/framework"
+	e2eskipper "k8s.io/kubernetes/test/e2e/framework/skipper"
+	imageutils "k8s.io/kubernetes/test/utils/image"
+
+	"github.com/onsi/ginkgo"
+)
+
+var _ = SIGDescribe("Downward API [Serial] [Disruptive] [NodeFeature:EphemeralStorage]", func() {
```
Comment (spiffxp) [inline test/e2e/common/storage/downwardapi.go:33]: Ah ok this was split out of the previous suite, that's why it's not showing as a move

**Review by spiffxp — APPROVED** [review]
/approve
/lgtm

**Comment by BenTheElder** [comment]
well that worked 🙃

**Pull request merged**